src/core/helpers.js

```diff
@@ -1252,39 +1252,61 @@ wb.escapeAttribute = function( str ) {
 
 /*
 * Find most common Personal Identifiable Information (PII) in a string and return either the cleaned string either true/false
-* @param {string} str
-* @param {boolean} toClean
+* @param{string} str (required)- the content that needs to be verified
+* @param {boolean} toClean - if true will scrub the content
+* @param {object} the 2nd param can also be an  object having the following properties:*
+* 	{boolean} bMarker (optional) - if true, will replace the scrubbed values with string "[REDACTED/CAVIARDÉ]"
+* 	{string} caseOnly (optional) - this param should match a key name from the "regEx" object. If present, it will search and replace only the values corresponding that pattern
+* 	{regex} customCase (optional) - this param is a regex. It will search and replace the values corresponding that pattern
+
 * @return {string | true | false}
 * @example
 * wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", true )
 * returns "email:, phone:",
+*
 * wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", false )
 * returns true
+*
+* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", { "bMarker":true, "caseOnly":"email" } )
+* returns "email:[REDACTED/CAVIARDÉ], phone:123 123 1234"
+*
+* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234, numéro de cas 12345678", { "bMarker":true, "customCase":/\b(?:case[\s-]?number[\s\-\\.]?(?:\d{5,10}))|(?:numéro[\s-]?de[\s-]?cas[\s\-\\.]?(?:\d{5,10}))/ig })
+* returns "email:[REDACTED/CAVIARDÉ], phone:[REDACTED/CAVIARDÉ], [REDACTED/CAVIARDÉ]"
 */
-wb.findPotentialPII = function( str, toClean ) {
-
-	if ( typeof str  !== "string" ) {
+wb.findPotentialPII = function( str, opts ) {
```

Comment:
Example d'idée

option (2e paramètre) par défaut lorsque boolean ou undefined.

```js
{
  digits: 1,
  passport: 1,
  email: 1,
  postalCode: 1,
  username: 1,
  password: 1
}
```

Custom parameter like:
```js
{
   email: /kekchose/gi,
   digits: 1
   custom1: /kekchosedautre/gi
}
```

Iterate to build all the regex to execute with a for...in statement

```js
{
   email: /kekchose/gi,
   digits: 1
   custom1: /kekchosedautre/gi,
   replaceWithMarker: 1,
   toClean: 1 // if true replace if false or undefine don't replace
}
```

Reply:
As discussed, the function will have 3 params:

1. `str ` (required) - the string that needs to ne verified;
2. `scope` if passed as Boolean is required as true if the content needs to be scrubbed; if passed as object is optional. 
3. `opts` (optional) - object that will customize the way the content is replaced;